packages/cli/src/utils/deploy.ts

```diff
@@ -17,12 +16,27 @@ import { getRegisterTableCallData } from "./tables/getRegisterTableCallData";
 import { getTableIds } from "./tables/getTableIds";
 import { confirmNonce } from "./utils/confirmNonce";
 import { deployContract } from "./utils/deployContract";
-import { fastTxExecute } from "./utils/fastTxExecute";
+import { fastTx } from "./utils/fastTxExecute";
```

Comment:
Might see if we can use our internal createContract helper, since it has its own nonce tracking

Reply:
I did originally try that but came across issues when it was sending a bunch of txs. I could have been doing something wrong though so would definitely be interested to see how to do it correctly.

Reply:
oh interesting! thanks for trying, happy to go with this and revisit in the future